tests/flow/array-filter/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -41,13 +41,15 @@ console.log(filteredItems);
 /* @flow */
 
 function filterItems(items: Array<string | number>): Array<string | number> {
-  return items.map(item => {
-    if (typeof item === \"string\") {
-      return item.length > 2 ? item : null;
-    } else {
-      return item * 10;
-    }
-  }).filter(Boolean);
+  return items
+    .map(item => {
+      if (typeof item === \"string\") {
+        return item.length > 2 ? item : null;
+      } else {
+        return item * 10;
+      }
+    })
+    .filter(Boolean);
```

Comment:
This is your only concern that has not been addressed. I think while it may not be ideal, it is a decent way to print it.